src/pages/organize/[orgId]/journeys/[journeyId]/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { GetServerSideProps } from 'next';
+import Head from 'next/head';
+
+import AllJourneyInstancesLayout from 'layout/organize/AllJourneyInstancesLayout';
+import { ColumnNames } from 'components/journeys/JourneyInstancesDataTable/getColumns';
+import getOrg from 'fetching/getOrg';
+import JourneyInstancesDataTable from 'components/journeys/JourneyInstancesDataTable';
+import { PageWithLayout } from 'types';
+import { scaffold } from 'utils/next';
+import { TagMetadata } from 'pages/api/organize/[orgId]/journeys/[journeyId]/getTagMetadata';
+import ZetkinQuery from 'components/ZetkinQuery';
+import { journeyInstancesResource, journeyResource } from 'api/journeys';
+import { ZetkinJourney, ZetkinJourneyInstance } from 'types/zetkin';
+
+const scaffoldOptions = {
+  authLevelRequired: 2,
+  localeScope: ['layout.organize', 'misc.breadcrumbs', 'pages.organizeJourney'],
+};
+
+export const getServerSideProps: GetServerSideProps = scaffold(async (ctx) => {
+  const { orgId, journeyId } = ctx.params!;
+
+  await ctx.queryClient.prefetchQuery(
+    ['org', orgId],
+    getOrg(orgId as string, ctx.apiFetch)
+  );
+  const orgState = ctx.queryClient.getQueryState(['org', orgId]);
+
+  const { state: journeyQueryState } = await journeyResource(
+    orgId as string,
+    journeyId as string
+  ).prefetch(ctx);
+
+  if (
+    orgState?.status === 'success' &&
+    journeyQueryState?.status === 'success'
+  ) {
+    return {
+      props: {
+        journeyId,
+        orgId,
+      },
+    };
+  } else {
+    return {
+      notFound: true,
+    };
+  }
+}, scaffoldOptions);
+
+type JourneyInstancesOverviewPageProps = {
+  journeyId: string;
+  orgId: string;
+};
+
+const JourneyInstancesOverviewPage: PageWithLayout<
+  JourneyInstancesOverviewPageProps
+> = ({ orgId, journeyId }) => {
+  const journeyQuery = journeyResource(orgId, journeyId).useQuery();
+  const journeyInstancesQuery = journeyInstancesResource(
+    orgId,
+    journeyId
+  ).useQuery();
+  const journey = journeyQuery.data as ZetkinJourney;
+
+  return (
+    <>
+      <Head>
+        <title>{journey.plural_name}</title>
+      </Head>
+      <ZetkinQuery queries={{ journeyInstancesQuery }}>
+        <JourneyInstancesDataTable
+          {...{
+            journey,
+            ...(journeyInstancesQuery.data as {
+              columnNames: ColumnNames;
+              journeyInstances: ZetkinJourneyInstance[];
+              tagMetadata: TagMetadata;
+            }),
+          }}
+        />
+      </ZetkinQuery>
+    </>
+  );
+};
+
+JourneyInstancesOverviewPage.getLayout = function getLayout(page) {
+  return <AllJourneyInstancesLayout>{page}</AllJourneyInstancesLayout>;
```

Comment:
I think `fixedHeight` should be true on this page, so that the table occupies the entire space (and the header stays put as you scroll).

Right now, the scrolling in the table doesn't seem to work at all. Even though there are a lot more journeys than what fits on one page (made evident when I sort on any column), the page looks like this for me, and doesn't scroll vertically:

![image](https://user-images.githubusercontent.com/550212/161042413-4a26561c-a002-4417-aead-a8cdc9673e81.png)

Horizontal scrolling works fine though.

I think this should work the way the view data table works. I'm not sure exactly what settings make it work the way it does though. I can help research it if it's not obvious when you look at it.

Reply:
agree and done